webserver/views/api.py

```diff
@@ -253,7 +254,17 @@ def _validate_listen(listen):
                 if len(tag) > MAX_TAG_SIZE:
                     _log_raise_400("JSON document may not contain track_metadata.additional_info.tags "
                                    "longer than %d characters." % MAX_TAG_SIZE, listen)
-
+        # MBIDs
+        lmbid = listen['track_metadata']['additional_info'].get('release_mbid')
+        if not is_valid_uuid(lmbid):
```

Comment:
It's probably better to check if this value is actually specified first (same with other MBID checks).